taxi/models.py

```diff
@@ -0,0 +1,31 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
```

Comment:
`name` and `model` 63? but `license_number` 255?) Why?

Reply:
Because for the name and model it is unlikely that a length will be indicated greater than 63. The length of license is not indicated in the task. So, I suggest that it can be very long

Reply:
but in the future, this length must be 8)

Reply:
I can't predict the future)

Reply:
)))